modules/benchmark/R/calc.metrics.R

```diff
@@ -12,44 +11,74 @@
 ##' 
 ##' @author Betsy Cowdery
 
-calc.metrics <- function(data.path, format, vars_used, model_run, metrics, start_year, end_year, site){
+calc.metrics <- function(data.path, format, model_run, metrics, start_year, end_year, site, bm, ens){
   
-  # Right now many argument are being read in as R list objects
-  # This won't work remotely but at the moment I don't have a good solution
+  # Right now I'm making the inappropriate assumption that storage type will be 
+  # empty unless it's a time variable. 
+  # This is because I haven't come up for a good way to test that a character string is a date format
```

Comment:
@mdietze revisiting this comment - the issue #888 is resolved, but I'm not entirely sure I need to change anything right now?